src/Apps/Settings/Routes/Shipping/Components/SettingsShippingAddressForm.tsx

```diff
@@ -157,31 +161,10 @@ export const SettingsShippingAddressForm: FC<SettingsShippingAddressFormProps> =
         handleChange,
         handleBlur,
         setFieldValue,
-        setFieldError,
         isValid,
         isSubmitting,
         submitForm,
       }) => {
-        // incorporate phone validation result into Formik values & errors
-        const handlePhoneNumberValidation = ({
-          isValid,
-          national,
-          region,
-        }: PhoneNumberValidationResult) => {
-          if (!isValid) {
-            setFieldValue("attributes.phoneNumber", "")
-            setFieldError(
-              "attributes.phoneNumber",
-              "Please enter a valid phone number"
-            )
```

Comment:
Actually, maybe it is easier to just set a field error rather than make an assertion about the schema being invalid. Will revisit tomorrow.